### PR TITLE
Feature/msdesc header button

### PR DESCRIPTION
**Pull request by saramaenza**

### CODE REVIEW

File: src/app/panels/text-panel/text-panel.component.ts [hunk 1]
```diff
@@ -66,6 +70,15 @@ export class TextPanelComponent implements OnInit, OnDestroy {
     shareReplay(1),
   );
 
+  public msDesc$ = this.evtModelService.editionSource$.pipe(
+    map(s => {
+      const msDesc = s.querySelector<HTMLElement>('msDesc');
+      const msDescParser = createParser(MsDescParser, parse);
+
+      return msDescParser.parse(msDesc);
```
Comment (szenzaro) [inline src/app/panels/text-panel/text-panel.component.ts:78]: This should be handled by a parser. Why is there parser logic inside a component?

File: src/app/panels/text-panel/text-panel.component.ts [hunk 2]
```diff
@@ -90,12 +103,24 @@ export class TextPanelComponent implements OnInit, OnDestroy {
...
+      this.secondaryContent = '';
+    }
+    else {
+      this.showSecondaryContent = false;
+      this.msDescOpen = false;
+    }
+  }
   getSecondaryContent(): string {
     return this.secondaryContent;
   }
```
Comment (szenzaro) [inline src/app/panels/text-panel/text-panel.component.ts:126]: No need of this, just access the class field.

File: src/app/panels/text-panel/text-panel.component.html [hunk 3]
```diff
@@ -25,7 +30,10 @@
                 [itemsToHighlight]="itemsToHighlight$ | async"></evt-page>
         </ng-container>
     </div>
-    <div secondary-content>{{ getSecondaryContent() }}</div>
+    <div secondary-content>
+        <evt-ms-desc *ngIf="msDescOpen" [data]="msDesc$ | async"></evt-ms-desc>
+        {{ getSecondaryContent() }}
```
Comment (szenzaro) [inline src/app/panels/text-panel/text-panel.component.html:35]: Just access the class field

**Review by szenzaro — CHANGES_REQUESTED** [review]
Missing CHANGELOG changes

File: src/app/panels/text-panel/text-panel.component.ts (hunk 1 above)
Comment (giuliac89) [reply src/app/panels/text-panel/text-panel.component.ts:78]: Parser logic should never be in a component. Move it to evt-model-service.

**Review by giuliac89 — CHANGES_REQUESTED** [review]

**Review by giuliac89 — APPROVED** [review]

**Pull request closed without merging**